jobs/rocoto/awips_20sh.sh

```diff
@@ -19,14 +19,15 @@ source "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
 status=$?
 (( status != 0 )) && exit "${status}"
 
-export job="awips"
+export job="awips_20sh.sh"
 export jobid="${job}.$$"
 
 # TODO (#1228) - This script is doing more than just calling a j-job
 #   Also, this forces us to call the config files here instead of the j-job
 source "${HOMEgfs}/ush/jjob_header.sh" -e "awips" -c "base awips"
```

Comment:
@KateFriedman-NOAA I addressed (2); I am not sure how to answer (1).

Reply:
@HenryWinterbottom-NOAA There should be new, separate config files and [MACHINE].env entries for each of the two new jobs. And really, probably separate config.resources entries as well.

Reply:
Before we end up making more configs, are they using the same resources; walltime, cores, etc.?

Reply:
If we use the resources from ops then they should be different resources in terms of walltime but same otherwise:

awips_20km_1p0:
```
#PBS -l walltime=00:10:00
#PBS -l select=1:ncpus=1:mem=3GB
```
https://github.com/NOAA-EMC/global-workflow/blob/dev/gfs.v16/ecf/scripts/gfs/atmos/post_processing/awips_20km_1p0/jgfs_atmos_awips_master.ecf#L6:L7

awips_g2:
```
#PBS -l walltime=00:05:00
#PBS -l select=1:ncpus=1:mem=3GB
```
https://github.com/NOAA-EMC/global-workflow/blob/dev/gfs.v16/ecf/scripts/gfs/atmos/post_processing/awips_g2/jgfs_atmos_awips_g2_master.ecf#L6:L7

Reply:
I left this same since the original script, `awips.sh` which was calling 2 j-jobs, was using a single configuration.

If this is still true, do we need extra configuration files?